presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -340,6 +356,11 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
           taskRuntimeStats[statName] = stat.second;
         }
       }
+
+      OP_STATS_TO_RUNTIME_STATS(spilledBytes, kBytes);
+      OP_STATS_TO_RUNTIME_STATS(spilledRows, kNone);
+      OP_STATS_TO_RUNTIME_STATS(spilledPartitions, kNone);
+
```

Comment:
We already have spill stats in OperatorStats. I think it might be easier to display it in runtime stats as well. We could also just leave it in OpStats. WDYT?

Reply:
I suppose they are being aggregated over all tasks, then there no point to have runtime stats, since we can see them as stats?